instagram/src/components/PostContainer/PostFooter.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+export default class PostFooter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: props.likes
+    }
+
+  }
+
+  addLikes = () => {
+    this.setState({
+      likes: this.state.likes + 1
+    })
```

Comment:
Also with the like functionality, your counter is currently increasing every time it is clicked. This means that the user can like the post more than once. How would you implement a like that will only increase 1 when clicked, then decrease when clicked again.